lib/core/form/components/form-renderer.component.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, ViewEncapsulation, Input } from '@angular/core';
+import { FormBaseComponent } from './form-base.component';
+
+@Component({
+    selector: 'adf-form-renderer',
+    templateUrl: './form-renderer.component.html',
+    styleUrls: ['./form-renderer.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class FormRendererComponent  {
+
+    @Input()
+    processedForm: FormBaseComponent;
```

Comment:
can we rename it in formDefinition ?

Reply:
Renamed to parsedFormDefinition as formDefinition created ambiguity.